src/apis/fs.cpp

```diff
@@ -98,12 +162,12 @@ static int fs_list(lua_State *L) {
 static int fs_exists(lua_State *L) {
     lastCFunction = __func__;
     const path_t path = fixpath(get_comp(L), checkstring(L, 1), true);
-    if (std::regex_search(path, pathregex(WS("^\\d+:")))) {
+    if (std::regex_search((*path.begin()).native(), pathregex("^\\d+:"))) {
```

Comment:
Note: This will break if a user mounts a real path `/\d+:/...`. However, this isn't possible on any system except Linux, and even then it requires root access to make the directory. I hope nobody runs into this. (I think an invalid result will just crash the computer, but more testing may be required to determine whether this is more unsafe.)

Reply:
Clarification: This doesn't affect root directories, but it affects directories in the CWD, so this can happen to any normal user. I'll add a check to block mounting real directories starting with `\d+:`.